problem_builder/templates/html/mentoring_with_steps.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+<div class="mentoring themed-xblock" data-step="{{ self.step }}">
+
+  {% if show_title and title %}
+  <div class="title">
+    <h2>{{ title }}</h2>
+  </div>
+  {% endif %}
+
+  <div class="assessment-question-block">
+    {{child_content|safe}}
```

Comment:
@itsjeyd nit: I believe 

    {{ for child_content in children_contents }}
        {{ children_content|safe}}
    {{ endfor }}

would be slightly more flexible: if needed, each child could be wrapped into a dom element, or otherwise post-processed in template